FlappyBird.py

```diff
@@ -1,3 +1,6 @@
+#amazing game 
+#install pygame and numpy before running 
```

Comment:
Rather than adding a comment can you add a `requirements.txt`
This can help you - https://blog.usejournal.com/why-and-how-to-make-a-requirements-txt-f329c685181e